docs/victoriametrics/changelog/CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ See also [LTS releases](https://docs.victoriametrics.com/victoriametrics/lts-rel
 
 * FEATURE: [alerts](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/rules): enhance alerting rule `DiskRunsOutOfSpaceIn3Days` and `NodeBecomesReadonlyIn3Days` to account for [deduplication](https://docs.victoriametrics.com/#deduplication) and [indexDB](https://docs.victoriametrics.com/#indexdb) growth when calculating disk consumption rate.
 * FEATURE: [dashboards/single](https://grafana.com/grafana/dashboards/10229), [dashboards/cluster](https://grafana.com/grafana/dashboards/11176): restore panel `Storage full ETA` using calculation logic from [#8955](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/8955). It was previously removed in [#8492](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/8492).
+* FEATURE: [vmselect](https://docs.victoriametrics.com/victoriametrics/cluster-victoriametrics/) in [VictoriaMetrics cluster](https://docs.victoriametrics.com/victoriametrics/cluster-victoriametrics/): dynamically adjusts the concurrent dial limit between 8 and 64 based on the `-search.maxConcurrentRequests`. Also, goroutines now have the chance to access available connections while awaiting the dial limit token.
```

Comment:
it looks to me like the change describes what was done without mentioning why it was done.
As a user, I much more prefer "why" to understand what the benefit this change gives to my use case. 
@makasim @jiekun would you mind checking it from the user PoV?

Reply:
agree

Reply:
Added the following description:
> This enables faster connection establishment when sudden requests arrive and reduces the blocking time during the availability check for connections.